docs/how-to/installation/docker.txt

```diff
@@ -56,29 +96,99 @@ master with the edited configuration:
 
 .. code::
 
-   docker run -v "$PWD"/master.yaml:/etc/determined/master.yaml determinedai/determined-master:VERSION
+   docker run \
+       -v "$PWD"/master.yaml:/etc/determined/master.yaml \
+       determinedai/determined-master:VERSION
+
+To start the master with environment variables instead of a configuration file:
+
+.. code::
+
+   docker run \
+       --name determined-master \
+       --network determined \
+       -p 127.0.0.1:8080:8080 \
+       -e DET_DB_HOST=determined-db \
+       -e DET_DB_NAME=determined \
+       -e DET_DB_PORT=5432 \
+       -e DET_DB_USER=postgres \
+       -e DET_DB_PASSWORD=<DB password> \
+       -e DET_HASURA_SECRET=<Hasura secret> \
+       -e DET_HASURA_ADDRESS=hasura:8080 \
+       determinedai/determined-master:VERSION
+
+Note that this references the PostgreSQL and Hasura containers running on
+``determined`` Docker network created above. If you are running PostgreSQL
+externally, specify the hostname for the PostgreSQL server in place of
+``determined-db``.
+
+Determined Agents
+~~~~~~~~~~~~~~~~~
+
+As is the case for the master, Determined agent :ref:`configuration values
```

Comment:
blocking: Add a target for the agent section of the config page and link directly there.

Reply:
added links for master and agent config sections